bibedit/record_conference.xml

```diff
@@ -22,11 +22,10 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
   <datafield tag="111" ind1=" " ind2=" ">
     <subfield code="a">VOLATILE:conference title</subfield>
     <subfield code="b">VOLATILE:subtitle</subfield>
-    <subfield code="d">VOLATILE:date from - to</subfield>
-    <subfield code="x">VOLATILE:start date</subfield>
+    <subfield code="x">VOLATILE:start date YYYY-MM-DD</subfield>
+    <subfield code="y">VOLATILE:end date YYYY-MM-DD</subfield>
     <subfield code="c">VOLATILE:city, country</subfield>
     <subfield code="e">VOLATILE:acronym</subfield>
-    <subfield code="g">VOLATILE:cnum Cyy-mm-dd[.n]</subfield>
```

Comment:
Because it's automatically generated by the system out of the date. 

-- 
Florian Schwennsen 
Deutsches Elektronen-Synchrotron DESY 
Building 01 Room O1.446 
phone: +49-40-8998-6190 

> From: "Samuele Kaplun" <notifications@github.com>
> To: "inspirehep/inspire" <inspire@noreply.github.com>
> Cc: "Florian Schwennsen" <florian.schwennsen@desy.de>
> Sent: Monday, 8 June, 2015 13:32:08
> Subject: Re: [inspire] bibedit: update record templates (#88)

> In bibedit/record_conference.xml :
> >      <subfield code="c">VOLATILE:city, country</subfield>
> >      <subfield code="e">VOLATILE:acronym</subfield>
> > -    <subfield code="g">VOLATILE:cnum Cyy-mm-dd[.n]</subfield>

> To be sure, why g disappears?

> —
> Reply to this email directly or view it on GitHub .